scripts/class/game.js

```diff
@@ -1 +1,27 @@
 // Class Game : gère la logique principale du jeu.
+
+class Game {
+  constructor(player, secretCode, numberTry) {
+    this.player = player;
+    this.secretCode = secretCode;
+    this.attemptCountPlayer = 0;
+    this.numberTry = numberTry;
+  }
+
+  checkStatus() {
```

Comment:
Le nom n'est pas bon, par convention, quand tu fais une méthode qui renvoi un booléen, il est bon de la préfixer par "is".
dans ce cas isCheckStatus, ne veut rien dire. Tu pourrais par contre nommer la méthode : "isGameContinue" par exemple